src/bus/lattice.rs

```diff
@@ -173,6 +218,214 @@ impl DistributedBus {
     }
 }
 
+pub(crate) fn controlplane_wildcard_subject(ns: Option<&str>) -> String {
+    format!("{}.{}.>", super::nsprefix(ns), CPLANE_PREFIX) // e.g. wasmbus.control.* or wasmbus.control.Nxxx.*
```

Comment:
`"{}.{}.>"` or `"{}.{}.*"`?

Reply:
Needs to be `.>` because we have both `wasmbus.control.*` and `wasmbus.control.Nxxxx.*` (all-hosts and 1-host targeting)

Reply:
OK, I just assumed it was a typo 😄.